build.gradle

```diff
@@ -217,7 +214,6 @@ dependencies {
 		'syslog4j-0.9.46'
 	)
 	// needed for syslog4j
-	implementation 'net.java.dev.jna:jna:3.4.0'
```

Comment:
so jna is not needed for syslog4j?

Reply:
Only if you're logging to syslog, which due to the other changes in this PR, auth no longer does

Reply:
I'm actually a bit surprised that it works given it gets imported here: https://github.com/syslog4j/syslog4j/blob/master/src/main/java/org/productivity/java/syslog4j/impl/unix/socket/UnixSocketSyslog.java#L10-L12